.github/workflows/emoji-server-check-workflow.yml

```diff
@@ -0,0 +1,45 @@
+name: "Emoji up-to-date check from server"
+on:
+  push:
+  schedule:
+    # The CRON time format is as follows:
+    # ---------------------------------------
+    # minutes | hours |  day  | month | day
+    #                  (month)         (week)
+    # ---------------------------------------
+    # For more information on time formats, see https://crontab.guru/
+    # 
+    # The below line suggests that the event will be triggered everyday
+    # at 22:00 UTC times.
+    - cron:  '0 22 * * *'
+jobs:
+  check:
+    name: Emoji up-to-date check from server
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install dependencies
+        run: pip install -e .
+      - name: Validate python version
+        run: python --version
+      - name: Fetch emojis from server
+        run: ./tools/fetch-unicode-emoji-data
+      - name: Convert emojis dict to list
+        run: ./tools/convert-unicode-emoji-data
+      - name: Notify if emojis are not up-to-date
+        run: |
+          git diff --exit-code zulipterminal/unicode_emojis.py
+          if [ $? -eq 0 ] ; then
+            echo "Emojis up-to-date"
+          else
+            curl -X POST https://chat.zulip.org/api/v1/messages \
+              -u ${{ secrets.ZT_BOT_EMAIL }}:${{ secrets.ZT_BOT_API_KEY }} \
+              --data-urlencode type=stream \
+              --data-urlencode 'to=zulip-terminal'\
+              --data-urlencode 'subject=Emoji sync with server' \
+              --data-urlencode 'content=Heads up :caution:!
+                                        Emoji data needs to be updated.'
```

Comment:
We can easily change these later, but:
* I expect it'd be better to have a more general single topic for CI feedback, since we have everything in one stream.
* Re the content, we may wish to specify more that this originates from a server emoji update.

Reply:
- Re topic, maybe something like: "CI action feedback"?
- Re the content, the below message could be a potential description?:
          "Heads up! Emoji sync with the server failed!
           This originated from an update in server emoji. Kindly re-generate `unicode_emojis.py`"